CDCgov/prime-reportstream
# 7564 HL7/FHIR NK1 Mapping Updates

This PR updates the HL7/Patient translation configurations based on updates to the documentation.

The proposed changes are provided here: [HL7 to FHIR - NK1 Segment.xlsx](https://github.com/CDCgov/prime-reportstream/files/10322226/HL7.to.FHIR.-.NK1.Segment.xlsx)

Note: The diff for some of the HL7 files is messed up. I pushed the changes as separate commits on those and can show the diff as needed.

Test Steps:
1. Build this branch and run the unit and integration tests. Verify they all pass. Alternatively you can verify the Github Actions build passes. Note the HL7 to FHIR to HL7 integration test data was not changed, so the result of our HL7 to FHIR to HL7 conversion remains the same = no data changes.
2. Optional: Add business phone data on NK1.6 and check that the data translate to FHIR and to HL7 losslessly.

## Changes
- Updated Patient contact translations to handle the Work number separately
- Updated related tests

## Linked Issues
- #7564
